lib/grp.gi

```diff
@@ -573,6 +577,32 @@ InstallImmediateMethod( IsSolvableGroup, IsGroup and HasSize, 10,
     TryNextMethod();
     end );
 
+InstallMethod( IsSolvableGroup,
+    "if group size is known and is 2m or p^a q^b",
+    [ IsGroup and HasSize ], 25,
+    function( G )
+    local size;
+    size := Size( G );
+    if IsInt( size ) then
+      if size mod 4 <> 0 then
+        return true;
+      else
+        size := size/4;
+        while size mod 2 = 0 do
```

Comment:
This will only ever catch 2^a q^b, but not p^aq^b as claimed

Reply:
If a group has size p^a q^b for odd p and q, then it is already odd, hence is already caught by the Feit-Thompson test. Otherwise one of the primes is 2, and the size looks like 2^a q^b.

Reply:
Thanks. The installation text had confused me.

Reply:
Good point. Rewrote the text in the method.